TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/models/backfill_rfs.py

```diff
@@ -185,6 +219,26 @@ def _get_detailed_status(self) -> Optional[str]:
 
         return "\n".join([f"Shards {key}: {value}" for key, value in values.items() if value is not None])
 
+def get_working_state_index_backup_path(archive_dir_path: str = None) -> str:
+    shared_logs_dir = os.getenv("SHARED_LOGS_DIR_PATH", None)
+    if archive_dir_path:
+        backup_dir = archive_dir_path
+    elif shared_logs_dir is None:
+        backup_dir = "./working_state"
```

Comment:
make this more specific, like backfill_working_state

Reply:
Done